corehq/apps/export/conversion_mappings.py

```diff
@@ -42,7 +42,7 @@
 
 CASE_PROPERTY_MAPPING = {
     ("id", None): ([PathNode(name="number")], None),
-    ("_id", None): ([PathNode(name='caseid')], None),
+    ("_id", None): ([PathNode(name='_id')], None),
```

Comment:
see this: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/export/system_properties.py#L252